.github/workflows/python.yml

```diff
@@ -223,16 +215,6 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: "3.12"
-    - name: create s3
-      run: aws s3 mb s3://lance-integtest --endpoint http://localhost:9000
-    - name: create ddb
-      run: |
-        aws dynamodb create-table \
-          --table-name lance-integtest \
-          --attribute-definitions '[{"AttributeName": "base_uri", "AttributeType": "S"}, {"AttributeName": "version", "AttributeType": "N"}]' \
-          --key-schema '[{"AttributeName": "base_uri", "KeyType": "HASH"}, {"AttributeName": "version", "KeyType": "RANGE"}]' \
-          --provisioned-throughput '{"ReadCapacityUnits": 10, "WriteCapacityUnits": 10}' \
-          --endpoint-url http://localhost:8000
```

Comment:
Moved this into test too, so we can do automatic setup and tear down. Make local testing much easier.